src/room/useGroupCall.ts

```diff
@@ -395,13 +458,20 @@ export function useGroupCall(groupCall: GroupCall): UseGroupCallReturnType {
       console.error(error);
       updateState({ error });
     });
+
+    groupCallOTelMembership?.onJoinCall();
   }, [groupCall, updateState]);
 
-  const leave = useCallback(() => groupCall.leave(), [groupCall]);
+  const leave = useCallback(() => {
+    groupCallOTelMembership?.onLeaveCall();
+    groupCall.leave();
```

Comment:
Can it be made sure that when someone forcefully closes the window, this still gets called?

Reply:
Done